osquery/core/watcher.cpp

```diff
@@ -661,4 +654,4 @@ size_t getWorkerLimit(WatchdogLimitType name) {
   }
   return kWatchdogLimits.at(name).normal;
 }
-} // namespace osquery
+} // namespace osquery
```

Comment:
Please just make sure you have a new-line at eof, otherwise the linter is going to complain.